blackbox/docs/admin/ssl.rst

```diff
@@ -187,7 +187,7 @@ information including the different SSL modes please visit the
 .. _psql documentation: https://www.postgresql.org/docs/current/static/app-psql.html
 
 
-Setting up a Keystore/Truststore With a Certificate Chain
+Setting up a keystore/truststore with a certificate chain
```

Comment:
```suggestion
Setting up a Keystore/Truststore with a certificate chain
```

(proper nouns)

Reply:
This forces us to also amend all occurrences within the texts.

Reply:
Done, yet I will have to go through all the texts to amend the proper nouns accordingly. Will include the changes in a separate PR 'fixups'.